src/plugins/expressions/common/expressions/types/functions.ts

```diff
@@ -42,6 +42,8 @@ export interface ExpressionFunction<Name extends string, Context, Arguments, Ret
 
 // TODO: Handlers can be passed to the `fn` property of the Function.  At the moment, these Functions
 // are not strongly defined.
-interface FunctionHandlers {
+export interface FunctionHandlers {
   [key: string]: (...args: any) => any;
 }
+
+export type AnyExpressionFunction = ExpressionFunction<any, any, any, any>;
```

Comment:
hmmm this looks weird to me, is this a standard pattern ? couldn't we define all those arguments just as ExpressionFunction ?

Reply:
I don't know if I could call it a pattern, we just need in few places a type which corresponds to *some* expression function, but we don't exactly know&mdash;and are not interested&mdash;in what internal shape it has.

We cannot just use `ExpressionFunction` because it requires generic parameters `ExpressionFunction<?, ?, ?, ?>`. We could use `ExpressionFunction<string, any, any, any>` everywhere we need a general expression function type, but that is what this type alias is for.

I cannot use something like `ExpressionFunction<string, unknown, unknown, unknown>` as that breaks a lot of code throughout Kibana.